src/index.js

```diff
@@ -18,7 +18,8 @@ class App extends React.Component {
       mediaUrl: null,
       isTextEditable: true,
       sttType: 'bbckaldi',
-      analyticsEvents: []
+      analyticsEvents: [],
+      fileName: 'Kate Darling Ted Talk'
```

Comment:
Not directly related to this PR - but I was thinking this demo app/wrapper around the component might actually make more sense in a different folder (ie. it shouldn't be in `/src/`, but a parallel folder called `demo` or something).

Similarly, `sample-data` might not belong in `/src`.

Reply:
ok, to try this out I moved `sample-data` outside of `src` and I get this error:

```
Module not found: You attempted to import ../sample-data/KateDarling_2018S-bbc-kaldi.json which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
```

See `create-react-app` docs for more details 

https://facebook.github.io/create-react-app/docs/folder-structure

Also see our [README#system-architecture](https://github.com/bbc/react-transcript-editor#system-architecture.